src/index.js

```diff
@@ -1,9 +1,9 @@
 require('babel-polyfill');
 
 const Resource = require('./resource');
-const UserResource = require('./user.resource');
+// const UserResource = require('./user.resource');
```

Comment:
What you doing commenting out my code...

Reply:
I thought we should remove it from the index until it is finished so people don't start using it.